### PR TITLE
Upload config file on cartridge

### PR DESCRIPTION
**Pull request by OlgaYakovleva**
Installed cypress-file-upload
Made setups to allow files upploading
Created configuration-file-page.spec.js with test for successful and unsuccessful config.yml uploading

Part of #351

### CODE REVIEW

**Review by rosik — CHANGES_REQUESTED** [review]
1. I suggest replacing zip file with a more simple yaml, still invalid. Name them appropriately:

`config.good.yml`:
```yml
custom_section: {}
```

`config.bad.yml`:

```yml
topology: {}
# will be rejected:
# uploading system section "topology" is forbidden
```

2. Megre two test functions (`it()`). There's no need to repeat the same `test_uploadZone` assertions twice.

File: webui/cypress/integration/configuration-file-page.spec.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          alias = 'test-replicaset',
+          roles = {},
+          servers = {{http_port = 8080}},
+        }}
+      })
+
+      _G.cluster:start()
+      return true
+    `}).should('deep.eq', [true]);
+    });
```
Comment (rosik) [inline webui/cypress/integration/configuration-file-page.spec.js:23]: Indentation

**Comment by rosik** [comment]
And, please, provide a better pull request description

File: webui/cypress/fixtures/files/config.bad.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+topology: {}
```
Comment (rosik) [inline webui/cypress/fixtures/files/config.bad.yml:1]:
```suggestion
topology: {}
# Uploading this config will fail with the error:
# "uploading system section is forbidden"

File: webui/cypress/integration/configuration-file-page.spec.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    ///////////////////////////////////////////////////////////////////
+    cy.log('Upload conf file');
+    ////////////////////////////////////////////////////////////////////
+    let filepath = 'files/config.good.yml';
+    cy.get('input[type="file"]').attachFile(filepath);
+    cy.get('[data-cy="test_uploadZone"]').contains('New configuration uploaded successfully.');
+    cy.get('[data-cy="test_uploadZone"]').contains('config.good.yml');
+
+    ///////////////////////////////////////////////////////////////////
+    cy.log('Upload conf file not in yml format');
```
Comment (rosik) [inline webui/cypress/integration/configuration-file-page.spec.js:51]: The message is outdated

File: webui/cypress/integration/configuration-file-page.spec.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    cy.get('[data-cy="test_uploadZone"]').contains('Upload configuration');
+    cy.get('[data-cy="test_uploadZone"]').contains('New configuration can be uploaded here.');
+    cy.get('[data-cy="test_uploadZone"]').contains('Click or drag file to this area to upload');
+    cy.get('.meta-test__DownloadBtn').should('exist');
+
+    ///////////////////////////////////////////////////////////////////
+    cy.log('Upload conf file');
+    ////////////////////////////////////////////////////////////////////
+    let filepath = 'files/config.good.yml';
+    cy.get('input[type="file"]').attachFile(filepath);
```
Comment (rosik) [inline webui/cypress/integration/configuration-file-page.spec.js:46]:
```suggestion
    cy.get('input[type="file"]').attachFile('files/config.good.yml');
```

File: webui/cypress/integration/configuration-file-page.spec.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    cy.get('[data-cy="test_uploadZone"]').contains('New configuration uploaded successfully.');
+    cy.get('[data-cy="test_uploadZone"]').contains('config.good.yml');
+
+    ///////////////////////////////////////////////////////////////////
+    cy.log('Upload conf file not in yml format');
+    ////////////////////////////////////////////////////////////////////
+    filepath = 'files/config.bad.yml';
+    cy.get('input[type="file"]').attachFile(filepath);
```
Comment (rosik) [inline webui/cypress/integration/configuration-file-page.spec.js:54]:
```suggestion
    cy.get('input[type="file"]').attachFile('files/config.bad.yml');
```

File: webui/cypress/support/commands.js [hunk 6]
```diff
@@ -23,3 +23,4 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+import 'cypress-file-upload';
```
Comment (rosik) [inline webui/cypress/support/commands.js:26]: Missing newline

Comment (rosik) [reply webui/cypress/support/commands.js:26]: Is this file appropriate for importing modules?

**Review by rosik — COMMENTED** [review]
Please, revise the commits. Your pull request includes two cherry-picked from master. Instead, you should rebase on top of it.

File: webui/cypress/integration/configuration-file-page.spec.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        replicasets = {{
+          alias = 'test-replicaset',
+          roles = {},
+          servers = {{http_port = 8080}},
+        }}
+      })
+
+      _G.cluster:start()
+      return true
+    `
```
Comment (rosik) [inline webui/cypress/integration/configuration-file-page.spec.js:22]:
```suggestion
      `
```

File: webui/cypress/support/commands.js (hunk 6 above)
Comment (OlgaYakovleva) [reply webui/cypress/support/commands.js:26]: This import has been done according to instructions to work with upload module

**Review by rosik — APPROVED** [review]

**Pull request merged**